deploy/cluster-manager/chart/cluster-manager/templates/cluster_role.yaml

```diff
@@ -110,9 +111,10 @@ rules:
 - apiGroups: ["certificates.k8s.io"]
   resources: ["signers"]
   verbs: ["approve", "sign"]
+# TODO remove the create when the grpc impersonation is implemented
 - apiGroups: ["cluster.open-cluster-management.io"]
   resources: ["managedclusters"]
-  verbs: ["get", "list", "watch", "update", "patch"]
+  verbs: ["get", "list", "watch", "create", "update", "patch"]
```

Comment:
why add create verb? grpc server should have a separated rbac

Reply:
this will be removed when the grpc impersonation is implemented

Reply:
my understanding is this clusterrole is not for grpc server, why additional rule is needed.

Reply:
the cluster-manager operator need to create the clusterrole for grpc server, so the cluster-manager need the same permissions with the grpc server clusterrole, or else, the operator cannot create the clusterrole

Reply:
as we discussed, we will not leverage the impersonation